devtools/conda_install_reqs.sh

```diff
@@ -36,7 +36,7 @@ WORKAROUNDS=""
 REQUIREMENTS=`python ${DEVTOOLS_DIR}/setup_cfg_reqs.py`
 TESTING=`python ${DEVTOOLS_DIR}/setup_cfg_reqs.py --extra test`
 INTEGRATIONS=`cat ${DEVTOOLS_DIR}/tested_integrations.txt | tr "\n" " "`
-EXPERIMENTAL=`cat ${DEVTOOLS_DIR}/experimental_reqs.txt | tr "\n" " "`
+EXPERIMENTAL=$(python ${DEVTOOLS_DIR}/setup_cfg_reqs.py --extra simstore)
```

Comment:
Hardcoding the name of the extra in this file feels wrong.
For the current state of the code I understand that this would work, but for future maintainability it might be nice to rename the `extra` `simstore` to `experimental` and don't give users the options to choose which of the experimental feature dependencies they install, just make it all or nothing

Or we could make a catch all `extra` using [this SO answer](https://stackoverflow.com/a/70983864)